### ISSUE TITLE
How to restart training model

### ISSUE THREAD

**Issue by fangrh** [mzjb/DeepH-pack#60]
Hi,

When I run the example of using ABACUS, everything gone well except the training is canceled by the slurm system due to the time limit.

Because the maximum time of using the GPU accelerate card is limited for one submit in our cluster. I wander if I can restart the training when next submit ?

**Comment by mzjb** [comment]
Please refer to `resume` in https://deeph-pack.readthedocs.io/en/latest/keyword/train.html#:~:text=best_state_dict.pkl.-,resume,-%3A%20The%20path%20to.

**Comment by fangrh** [comment]
OK, it works, thank you very much. But the epoch will count from beginning. However, in the train.ini, the epoch is set 5000. 
```
[train]
epochs = 5000
train_ratio = 0.6
val_ratio = 0.2
test_ratio = 0.2
revert_decay_epoch = [600, 2000, 3000, 4000]
revert_decay_gamma = [0.4, 0.5, 0.5, 0.4]
```
Then the training will do around and around. What is the justification that the training is sufficient?

**Comment by mzjb** [comment]
Generally, an MSE loss accuracy to the order of 10^-6 is sufficient, but you can also monitor the actual precision based on the system you are studying to determine when to stop training.
